code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -591,7 +591,7 @@
 
 /obj/item/projectile/bullet/heavy_rifle_408/lethal
 	name = "hollow-point bullet"
-	damage_types = list(BRUTE = 20)
+	damage_types = list(BRUTE = 24)
 	armor_divisor = 0.7
 	wounding_mult = WOUNDING_WIDE
```

Comment:
At 0 armor: FMJ 30 * 1.5 = 45dmg
HP: 24 * 1.5 = 36 dmg

Might want to adjust HP to be better than FMJ against unarmored